day2.md

```diff
@@ -0,0 +1,171 @@
+
+I have to apologize that we didn't take many great pictures in the fish market because it was fairly crowded and all the foods there were screaming for our attentions.
+
+<img src="day2/IMG_20160906_101301.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_102331.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_102433.jpg" width="500"><br/>
+Here comes the first thing I got: my all time favorite tamagoyaki（玉子焼き）！
+
+<img src="day2/IMG_20160906_103227.jpg" width="500"><br/>
+<img src="day2/14409662_1115752155141144_7886916382605174951_o.jpg" width="500"><br/>
+<img src="day2/14444831_1115752548474438_2131506267391325625_o.jpg" width="500"><br/>
+<img src="day2/14435156_1115752615141098_910067847601582150_o.jpg" width="500"><br/>
+Then comes scallop sautéed in butter.
+
+<img src="day2/14409862_1115752071807819_502330450151416461_o.jpg" width="500"><br/>
+There are couple small sanding tables in the shop and we just ate there right after it was cooked!<br/>
+I wished I had tried out those fresh sashimi there. I saw someone had tuna cubes with the middle parts still icy.
+
+After enough wondering and calculation, we picked<sup>[2](#pick)</sup> a small sushi shop and ate there.
+<img src="day2/IMG_20160906_110423.jpg" width="500"><br/>
+The "restaurant" was very small. We sat on the other side of it, but that's pretty much it. Most "restaurants" in the midst of the market were about the same size. For some reason, my brain just connect these type of shops with adjectives such as "authentic" or "local"<sup>[3](#small)</sup>.
+
+<img src="day2/IMG_20160906_104720.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_105812.jpg" width="500"><br/>
+Here comes our sashimi bowel and salmon sashimi!<br/>
+They were good but a bit pricy and didn't live up to our expectations.
+
+<img src="day2/IMG_20160906_111430.jpg" width="500"><br/>
+We also grabbed some of these before leaving the market!😄<br/>
+One of my favorite snacks in Japan, I miss them!
+
+We then headed to  <a name="skytree">[Tokyo Skytree Town](http://www.tokyo-solamachi.jp/english/floor/)</a>!
+<br/><img src="day2/IMG_20160906_120835.jpg" width="500"><br/>
+<br/><img src="day2/IMG_20160906_121403.jpg" width="500"><br/>
+First thing we saw after we got out from the station was a store for this famous banana bread.
+
+The bottom floors of the skytree tower includes various areas such as a shopping district, a fashion zone, and a food court.
+
+We randomly walked into a manga/anime shop packed with manga and related products! <br />
+Grown up watch anime, the place recalled a lot of good old time for me.
```

Comment:
- "Grown up **watching** anime" or maybe "**Growing** up **watching** anime"
- "good old **times**" is the common saying